op-node/cmd/main.go

```diff
@@ -104,6 +101,13 @@ func RollupNodeMain(ctx *cli.Context) error {
 		return err
 	}
 
+	// Only pretty-print the banner if it is a terminal log. Other log it as key-value pairs.
+	if logCfg.Format == "terminal" {
+		log.Info("rollup config:\n" + cfg.Rollup.Description(chaincfg.L2ChainIDToNetworkName))
+	} else {
+		cfg.Rollup.LogDescription(log, chaincfg.L2ChainIDToNetworkName)
+	}
+
```

Comment:
To avoid duplicate code I'm fine with just always calling `LogDescription` instead

Reply:
Is this something that you want to do?